crt_portal/cts_forms/sorts.py

```diff
@@ -44,6 +44,25 @@ def report_sort(sort):
     return sort_exprs, sort
 
 
+def resource_sort(sort):
```

Comment:
This is basically the same as the method above it - is there a way to DRY this up?